### DISCUSSION TITLE
How in the HELL is there not a Export to CSV button standard on a Table of Issues for a Project

### DISCUSSION THREAD

**Discussion by cjmiller81** [community/community#143712] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

What century are we living in where there is not a standard Export to CSV option for a table of issues for a Project in Git Hub???
![image](https://github.com/user-attachments/assets/debed30c-55ba-42ed-8818-bdda3d7f8f09)

**Comment by Mobinshahidi** [comment]
If you’d like GitHub to consider adding an 'Export to CSV' feature, here’s how to make the suggestion:

1. Visit the [GitHub Feedback Discussions page](https://github.com/orgs/community/discussions) and start a new topic under 'Product Feedback' or 'Ideas.'
2. Alternatively, you can send a feature request through [GitHub Support](https://support.github.com/contact/feedback).

Posting in the Feedback Discussion often gets quicker visibility with both the community and GitHub staff!
